src/develop.js

```diff
@@ -0,0 +1,4 @@
+let a = ("I hope i made it true");
```

Comment:
Not shure about "()" here. When we use string type, no need in "()". I belive we can simplify it to
```suggestion
let a = 'I hope i made it true';
```

Reply:
thanks for the tip